code/modules/mob/living/simple_animal/hostile/ooze.dm

```diff
@@ -39,21 +41,12 @@
 	ADD_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
 
 /mob/living/simple_animal/hostile/ooze/attacked_by(obj/item/I, mob/living/user)
-	if(!check_edible(I))
+	if(!eat_atom(I, TRUE))
```

Comment:
Could you change I to be a more descriptive variable name?

Reply:
@Watermelon914 those kind of changes are not really useful for procs like attacked_by, the template automatically uses this var.